core/src/test/java/io/questdb/std/datetime/millitime/DateFormatCompilerTest.java

```diff
@@ -107,6 +107,22 @@ public void testDayOneDigit() throws Exception {
         assertThat("dyyyy", "2014-01-03T00:00:00.000Z", "32014");
     }
 
+    @Test
```

Comment:
What's the intent of tests below ? 
They ignore day/week field and return start of month .

Reply:
Current behavior when parsing is to ignore the info generated by week of year / week of month as well as day of year, as obviously the week won't allow us to pinpoint a specific day based on the other information, in line with what day of week currently is doing. This simply is there to assert that parsing correctly skips the new flags